Engine/script/runtimescriptvalue.h

```diff
@@ -309,13 +310,63 @@ struct RuntimeScriptValue
         return !(*this == rval);
     }
 
+    // FIXME: find out all certain cases when we are reading a pointer and store it
+    // as 32-bit value here. There should be a solution to distinct these cases and
+    // store value differently, otherwise it won't work for 64-bit build.
```

Comment:
Frankly, I don't remember what this comment means and whether it is still applicable.